data.tf

```diff
@@ -37,6 +39,77 @@ data "aws_iam_policy_document" "s3" {
   }
 }
 
+####################################################################################
+### Organizational S3 bucket
+### Note=> Not providing cloudtrail name at `aws:SourceArn` will bring an error
+####################################################################################
+
+data "aws_iam_policy_document" "org_s3" {
+  version = "2012-10-17"
+  statement {
+    sid    = "AWSCloudTrailAclCheck20150319"
+    effect = "Allow"
+    principals {
+      type        = "Service"
+      identifiers = ["cloudtrail.amazonaws.com"]
+    }
+    actions   = ["s3:GetBucketAcl"]
+    resources = ["arn:aws:s3:::${local.bucket_name}"]
+    condition {
+      test     = "StringEquals"
+      variable = "aws:SourceArn"
+      values   = ["arn:aws:cloudtrail:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:trail/${local.trail_name}"]
```

Comment:
use local's for data sources, for example
`region = data.aws_region.current.name`

Reply:
`"arn:aws:cloudtrail:${local.region}:...`